CactusTutorial.ipynb

```diff
@@ -535,11 +545,117 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# start simulation, watch log output\n",
-    "!./simfactory/bin/sim create-run tov_ET \\\n",
+    "%%bash\n",
+    "# start simulation\n",
+    "./simfactory/bin/sim create-submit tov_ET \\\n",
     "  --parfile=par/tov_ET.par --procs=2 --num-threads=1 --walltime=0:20:0"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `submit` command submitted the simulation to the queueing system to run in the background asking for a maximum runtime of 20 minutes. On your laptop it will start right away, on a cluster the queuing system will wait until a sufficient number of nodes is able to start your simulation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can check the status of the simulation with the command below. You can run this command repeatedly until the job shows <pre>[ACTIVE (FINISHED)...</pre> as it's state. Prior to that, it may show up as QUEUED or RUNNING."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "bash: line 1: simfactory/bin/sim: No such file or directory\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "simfactory/bin/sim list-simulations tov)ET"
```

Comment:
There's a typo here. It should be "tov_ET" not "tov)ET"

Reply:
agreed. Will fix.